spynl/tests/test_log_error.py

```diff
@@ -143,3 +143,18 @@ def test_log_given_exc_type_and_msg(logger, fake_request):
 
         assert logger.error_log['msg'] == ("TEST Error of type Argh "
                                            "with message: 'what the hell'.")
+
+
+def test_log_debug_message(logger, fake_request):
+    """Test that the debug_message is logged."""
+
+    class SomeException(Exception):
+        debug_message = "Debug this."
+
```

Comment:
My idea was to implement it like this:
```python
class SomeException(Exception):
        def __init__(self):
            self.debug_message = 'Debug this.'
```

I tested it, and it doesn't seem to work this way. (This way the message and the debug message will be together)